### PR TITLE
14.0 Add module:  sale quotation template product multi add

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
Similar to **sale_product_multi_add**, but for Quotation Template (sale.order.template) model.

### CODE REVIEW

File: sale_quotation_template_product_multi_add/wizards/qt_add_products.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+from odoo import api, fields, models
+
+
+class QTAddProducts(models.TransientModel):
+    _name = "qt.add.products"
```
Comment (rousseldenis) [inline sale_quotation_template_product_multi_add/wizards/qt_add_products.py]: Better to use an explicit name

File: sale_quotation_template_product_multi_add/wizards/qt_add_products.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    _description = "Quotation Template Add Products"
+
+    products = fields.Many2many(comodel_name="product.product")
```
Comment (rousseldenis) [inline sale_quotation_template_product_multi_add/wizards/qt_add_products.py]: The convention would require 'product_ids' instead (same for here under)

File: sale_quotation_template_product_multi_add/wizards/qt_add_products_view.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_add_products_to_qt" model="ir.ui.view">
+        <field name="name">qt.add.products.form</field>
+        <field name="model">qt.add.products</field>
+        <field name="arch" type="xml">
+            <form string="Products">
+                <field name="products" />
```
Comment (rousseldenis) [inline sale_quotation_template_product_multi_add/wizards/qt_add_products_view.xml]: Sometimes it's better to add a short help text for user. What do you think ?

File: sale_quotation_template_product_multi_add/wizards/qt_add_products.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    items = fields.One2many(
+        comodel_name="qt.add.products.items",
+        inverse_name="wizard_id",
+    )
+
+    def create_items(self):
+        for wizard in self:
+            for product in wizard.products:
+                self.env["qt.add.products.items"].create(
```
Comment (rousseldenis) [inline sale_quotation_template_product_multi_add/wizards/qt_add_products.py]: It's better to use create multi functionality

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Some comments

File: sale_amount_payment_link/tests/test_sale_amount_payment_link.py [hunk 5]
```diff
@@ -1,11 +1,13 @@
 # Copyright 2021 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
+from odoo.tests import tagged
 from odoo.tests.common import Form
 
 from odoo.addons.account.tests.common import AccountTestInvoicingCommon
 
 
+@tagged("post_install")
```
Comment (rousseldenis) [inline sale_amount_payment_link/tests/test_sale_amount_payment_link.py:10]: @ilyasProgrammer Could you remove unrelated module modifications ?

Comment (rousseldenis) [reply sale_amount_payment_link/tests/test_sale_amount_payment_link.py:10]: Ok, I see, it's to fix tests. But we can do it in another PR in order to merge it fast. Could you do it ?

Comment (ilyasProgrammer) [reply sale_amount_payment_link/tests/test_sale_amount_payment_link.py:10]: ok

Comment (ilyasProgrammer) [reply sale_amount_payment_link/tests/test_sale_amount_payment_link.py:10]: https://github.com/OCA/sale-workflow/pull/1989

Comment (francesco-ooops) [reply sale_amount_payment_link/tests/test_sale_amount_payment_link.py:10]: @ilyasProgrammer rebase?

**Comment by ilyasProgrammer** [comment]
this PR was broken. Moved here https://github.com/OCA/sale-workflow/pull/1990
